### ISSUE TITLE
Accordion Panel Heading Color

### ISSUE THREAD

**Issue by DevQueen** [angular-ui/bootstrap#4420]
Is there a way to change the color of an accordion panel heading if it is open?  I Can change ALL of the panel heading colors, but I have been unable to find any resources supporting changing just one at a time.

**Comment by icfantv** [comment]
In the future, please do not post support requests in the issues forum.  Rather, please follow the instructions [here](https://github.com/angular-ui/bootstrap#need-help).

By default, the class applied to the `accordion-group` is `panel-default`, but you can override this via the `panel-class` attribute as documented [here](http://angular-ui.github.io/bootstrap/#/accordion).

**Comment by DevQueen** [comment]
I apologize.  I’m pretty new to all of this.  The last time I tried to ask a question on Github (on another feature), I was told to “open an issue”.  I keep getting chastised it seems because everyone on GitHub has a different set of rules.  Again, I apologize.

The link you sent me which allows me to “view” the panel-class, has no css included neither does the plunker, and I can’t find any documentation explaining how to use it at all.  I posted a question on StackOverFlow regarding the panel-class and received no responses.  I tried to use the webchat link, but couldn’t figure out how to enter a question.  I just kept seeing messages that “so-and-so has joined”, but I couldn’t see any chatting going on by anyone.  It just kept giving me JavaScript runtime popup errors.  If there is any actual “code” or “css” showing how to use the panel-class, I would appreciate your directing me to it.  I’m trying to convince my boss to use the Angular accordion rather than the JQuery he is using directly on the page.  He told me if I could get the accordion menu to look and perform the same that we could use it.  The only thing I am missing is the panel color change when a panel is open.  If you could give me a little more direction, it would be greatly appreciated.


[Beryl Small]

Beryl Small
IT / Programmer
985.629.5934 (D)
800.892.8179 (O)
985.892.8114 (F)
http://www.pecsafety.com/
This e-mail is confidential and is intended solely for the use of the recipient.  If you are not the intended recipient, be advised that you have received this e-mail in error and that any use, dissemination, forwarding, printing, or copying of this e-mail is strictly prohibited.

**Comment by icfantv** [comment]
I can't find your question on StackOverflow.  When did you post it?  Also, you need to realize that most OSS is developed during a developer's free time which means support can be, and typically is, slow.  This is usually where people get upset because we live in an "I need it right now" world.  OSS doesn't work like that.

The webchat works fine - if you're getting errors then it sounds like there are issues on your end.  Have you tried using a different browser?

The UI Bootstrap project has no built-in CSS (well, it has a few lines for a VERY specific use case), and uses the Bootstrap CSS classes.  `panel-default` is one such class.  `panel-class` is an attribute you specify an `<accordion-group>` element and it overrides the default `panel-default` class.  That is, you can specify your own CSS class (which lives in your own CSS code) or a different Bootstrap class.

If you want it to change color when it is open, you'll need to use `ng-class` and trigger on the same value as the `is-open` attribute as the markup accordion example shows.

**Comment by DevQueen** [comment]
My apologies again.  I wasn’t upset about how fast or slow you replied (I actually thought you replied pretty quickly).  I just seem to keep getting reprimanded for asking questions in the wrong place.  LOL.  No worries.  I posted the question on StackOverflow on yesterday.  Here is the link: http://stackoverflow.com/questions/32595125/angularjs-ui-accordion-change-panel-color-if-isopen

I was able to find the panel classes in boostrap.  I tried to use the one used in the example “panel-class=’panel-danger’” just to see if it would change the panel color, but it didn’t, so I figured I was missing something.  I do appreciate your help.

[Beryl Small]

Beryl Small
IT / Programmer
985.629.5934 (D)
800.892.8179 (O)
985.892.8114 (F)
http://www.pecsafety.com/
This e-mail is confidential and is intended solely for the use of the recipient.  If you are not the intended recipient, be advised that you have received this e-mail in error and that any use, dissemination, forwarding, printing, or copying of this e-mail is strictly prohibited.

**Comment by icfantv** [comment]
You need to look at the examples and read the HTML.  The `Delete account` accordion uses the `panel-class` attribute and the `I can have markup, too!` accordion shows how to use `ng-class` to selectively add a class based on whether or not the accordion is open.

**Comment by DevQueen** [comment]
Okay, thanks.  For whatever reason when I tried to use the panel-class element on the accordion group, the color did not change, so I figured I must be missing a step.  When I read the HTML, I did see a <script> on the top that said “ panel-class || panel-default”  I thought perhaps that was what I was missing, but even after adding that script to my page, tab colors remained the same.  I’ll keep trying.  I’ve come to find that one small little thing in HTML can cause features not to work as expected.

Thanks again for your help.


[Beryl Small]

Beryl Small
IT / Programmer
985.629.5934 (D)
800.892.8179 (O)
985.892.8114 (F)
http://www.pecsafety.com/
This e-mail is confidential and is intended solely for the use of the recipient.  If you are not the intended recipient, be advised that you have received this e-mail in error and that any use, dissemination, forwarding, printing, or copying of this e-mail is strictly prohibited.
